core/templates/dev/head/css/oppia.css

```diff
@@ -2162,9 +2164,8 @@ md-card.oppia-dashboard-tile {
 
 .oppia-dashboard-list-view-item {
   background: #fff;
-  margin: 20px auto 0;
-  padding: 10px 20px;
-  width: 100%;
+  margin: 20px 7.5px 0px 7.5px;
+  padding: 10px 12.5px;
```

Comment:
Okay, I will change it back to 20px :) 
Actually if I decreased the padding there would be more space for the content. But there isn't much difference. So I'll keep it at 20px.